examples/aws/network/example.tfvars

```
@@ -10,12 +10,14 @@ private_key_path = "./example_key"
 
 internal_domain = "internal.scalar-labs.com"
 
+multiple_public_key_folder_path = "./public_key"
```

```suggestion
public_key_folder_path = "./public_key"
```
since it does't have to have multiple keys.
Also, we should place it close to `public_key_path` since it is a similar setting.

---

Yes agree, will refactor

---

changed in https://github.com/scalar-labs/scalar-terraform/pull/93/commits/d0b2c6244ef2b6fa1caa9319f404f22e5b4f97f1